src/web/mjs/engine/Storage.mjs

```diff
@@ -795,17 +795,19 @@ export default class Storage {
      * LINUX: wxT("/\r\n\t");
      * WINDOWS: wxT("\\/:*?\"<>|\r\n\t");
      */
-    sanatizePath ( path ) {
+    sanitizePath ( path ) {
```

Comment:
This is a public method, if renamed all its usages must be renamed as well, otherwise the application stops to work correctly.

See:
- BookmarkConnector.mjs
- Manga.mjs
- Chapter.mjs